src/main/java/be/minelabs/mixin/ServerPlayNetworkHandlerMixin.java

```diff
@@ -0,0 +1,18 @@
+package be.minelabs.mixin;
+
+import net.minecraft.server.network.ServerPlayNetworkHandler;
+import net.minecraft.util.math.MathHelper;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.Unique;
+
+@Mixin(ServerPlayNetworkHandler.class)
+public abstract class ServerPlayNetworkHandlerMixin {
+    // Note: should stay consistent with variable in ClientPlayerInteractionManagerMixin
+    @Unique
+    private static final float SUBATOMIC_REACH_DISTANCE = 20f;
+
+    @Shadow @Final
+    public static double MAX_BREAK_SQUARED_DISTANCE = MathHelper.square(SUBATOMIC_REACH_DISTANCE);
```

Comment:
Doesn't this also effect the overworld/nether/end ?

Reply:
Probably yes. IIRC, this is the server side check to validate packets coming from the client. So as long as the client doesn't send invalid packets (reaching from too far) this shouldn't be an issue.

It could make hacking a client to fool the server easier, but if that ever proves to be an issue we can fix it.